After a quick run it seems like the CDI TCK also doesn't like this upgrade:

```
[ERROR] Failures: 
[ERROR]   ProcessBeanAttributesNotFiredForBuiltinBean>Arquillian.run:138->testProcessBeanAttributesNotFired:51 expected [true] but found [false]
[ERROR]   AfterTypeDiscoveryTest>Arquillian.run:138->testFinalAlternatives:128 expected [true] but found [false]
[ERROR]   AfterTypeDiscoveryTest>Arquillian.run:138->testInitialAlternatives:81 expected [3] but found [11]
[ERROR]   AfterTypeDiscoveryMassOperationsTest>Arquillian.run:138->testInitialAlternatives:77 expected [3] but found [11]
```

Maybe a Weld 5.0.1 is just too much for now? What we really need is a 5.0.0.SP3, which only changes the problematic Jakarta Annotation ManagedBean and absolutely nothing else. No comment, no whitespace, nothing. Just ManagedBean.

--- a/nucleus/parent/pom.xml
+++ b/nucleus/parent/pom.xml
@@ -137,7 +137,7 @@
         <!-- 3rd party dependencies -->
 
         <jboss.logging.annotation.version>2.2.1.Final</jboss.logging.annotation.version>
-        <jboss.logging.version>3.4.3.Final</jboss.logging.version>
+        <jboss.logging.version>3.5.0.Final</jboss.logging.version>
         <javassist.version>3.29.0-GA</javassist.version>
         <asm.version>9.3</asm.version>
         <jsch.version>0.1.56</jsch.version>

